src/components/Reservations.jsx/Reservations.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect } from 'react'
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchUserReservations } from '../../redux/reservations/reservationsSlice';
+
+
+function Reservations() {
+  const {reservations, loading} = useSelector(state => state.reservations)
+
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchUserReservations())
+  }, [])
+
+  const reservationsArr = reservations.products
+
+  if (loading) return <p>Loading...</p>
+
+  return (
+    <section className='bg-slate-200 w-[80vw] flex flex-col items-center overflow-x-auto'>
```

Comment:
- [ ] [Optional] Since the navbar already has a value for the width is supposed to use, for the other half of the screen you can use directly `w-full` tailwind class, in that way if you need to work in a future responsive design it would be something less to be worried about because the default behavior will be to use the all the available space.

Reply:
Good point 👍